### DISCUSSION TITLE
Page Not Showing Theme

### DISCUSSION THREAD

**Discussion by m0chan** [community/community#22872] [Other Feature Feedback, Questions, & Ideas]
<p>Hello All,</p>
<p>Apologies if this is a noob question but I really am lost here - I have created the following Repo <a href="https://github.com/m0chan/m0chan.github.io" rel="nofollow noopener">https://github.com/m0chan/m0chan.github.io</a> and enabled GitPages and if I visit <a href="http://m0chan.github.io" rel="nofollow noopener">m0chan.github.io</a> it appears as it should! Great! What’s the problem? </p>
<p>Well I have created a folder called ‘HackTheBox’ where I will store reports of boxes I have completed in markdown format. I would like to be able to visit <a href="http://m0chan.github.io/HackTheBox/Valentine.md" class="inline-onebox-loading" rel="nofollow noopener">m0chan.github.io/HackTheBox/Valentine.md</a> and it appears with the nice GitPage/Theme however it does not… It is displaying in plain text? </p>
<p>Any tips/pointers</p>
<p>m0chan</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>The file name is <a href="https://github.com/m0chan/m0chan.github.io/blob/master/HackTheBox/ValentineHTB.md" rel="nofollow noopener">ValentineHTB.md</a>, which is important :grinning: GitHub Pages converts your Markdown files to HTML. So if you want to see the <code>ValentineHTB.md</code> file on your GitHub Pages site, it is at <a href="https://m0chan.github.io/HackTheBox/ValentineHTB.html" rel="nofollow noopener">https://m0chan.github.io/HackTheBox/ValentineHTB.html</a> (note the <code>.html</code> extension instead of <code>.md</code>)</p>
<p>I hope that helps!</p>
